tools/render-test/cuda/cuda-compute-util.cpp

```diff
@@ -964,514 +988,611 @@ static SlangResult _compute(CUcontext context, CUmodule module, const ShaderComp
+                    auto shape = type->getResourceShape();
 
-                        auto baseShape = shape & SLANG_RESOURCE_BASE_SHAPE_MASK;
+                    auto baseShape = shape & SLANG_RESOURCE_BASE_SHAPE_MASK;
 
-                        switch (baseShape)
+                    switch (baseShape)
+                    {
+                        case SLANG_TEXTURE_1D: 
+                        case SLANG_TEXTURE_2D:
+                        case SLANG_TEXTURE_3D:
+                        case SLANG_TEXTURE_CUBE:
                         {
-                            case SLANG_TEXTURE_1D: 
-                            case SLANG_TEXTURE_2D:
-                            case SLANG_TEXTURE_3D:
-                            case SLANG_TEXTURE_CUBE:
-                            {
-                                RefPtr<CUDAResource> resource;
-                                SLANG_RETURN_ON_FAIL(CUDAComputeUtil::createTextureResource(entries[value->m_userIndex], typeLayout, resource));
-                                value->m_target = resource;
-                                break;
-                            }
-                            case SLANG_TEXTURE_BUFFER:
-                            {
-                                // Need a CUDA impl for these...
-                                // For now we can just leave as target will just be nullptr
-                                break;
-                            }
+                            RefPtr<CUDAResource> resource;
+                            SLANG_RETURN_ON_FAIL(CUDAComputeUtil::createTextureResource(entries[value->m_userIndex], typeLayout, resource));
+                            value->m_target = resource;
+                            break;
+                        }
+                        case SLANG_TEXTURE_BUFFER:
+                        {
+                            // Need a CUDA impl for these...
+                            // For now we can just leave as target will just be nullptr
+                            break;
+                        }
 
-                            case SLANG_BYTE_ADDRESS_BUFFER:
-                            case SLANG_STRUCTURED_BUFFER:
-                            {
-                                // On CPU we just use the memory in the BindSet buffer, so don't need to create anything
-                                RefPtr<MemoryCUDAResource> resource = new MemoryCUDAResource;
-                                SLANG_CUDA_RETURN_ON_FAIL(cuMemAlloc(&resource->m_cudaMemory, value->m_sizeInBytes));
-                                value->m_target = resource;
-                                break;
-                            }
+                        case SLANG_BYTE_ADDRESS_BUFFER:
+                        case SLANG_STRUCTURED_BUFFER:
+                        {
+                            // On CPU we just use the memory in the BindSet buffer, so don't need to create anything
```

Comment:
Looks like comment is out of date.

Reply:
Looks like it, yeah. That is code I copy-pasted during the refactoring, so I didn't even look at the logic inside the function.

I'm inclined to leave the stale comment in this PR so the diffs can in theory be more pure (not that git/GitHub diff is able to figure out the edits).